### PR TITLE
Add recipe for hilbertcurve

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This is ready for review, Windows failure is unrelated and as this is a noarch package and be safely ignored.

File: recipes/hilbertcurve/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (mbargull) [inline recipes/hilbertcurve/LICENSE]: xref: https://github.com/galtay/hilbertcurve/issues/20

**Comment by galtay** [comment]
hi all, I think I managed to get a new pypi release up (v1.0.3) with the LICENSE file included (also removed the :math: directive from the readme.rst so it would render) https://pypi.org/project/hilbertcurve/1.0.3/ 

does this mean folks will be able to conda install hilbertcurve?  if so, that rocks :)

File: recipes/hilbertcurve/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{% set name = "hilbertcurve" %}
+{% set version = "1.0.1" %}
```
Comment (galtay) [inline recipes/hilbertcurve/meta.yaml]: probably want 1.0.3 here now

**Comment by mbargull** [comment]
> does this mean folks will be able to conda install hilbertcurve? if so, that rocks :)

Indeed it does, thanks to @xhochy's addition!

File: recipes/hilbertcurve/meta.yaml (hunk 2 above)
Comment (mbargull) [reply recipes/hilbertcurve/meta.yaml]: Thanks, yes, an automatically generated PR has already been issued: https://github.com/conda-forge/hilbertcurve-feedstock/pull/1

**Pull request merged**